operations/helm/charts/mimir-distributed/CHANGELOG.md

```diff
@@ -28,6 +28,7 @@ Entries should include a reference to the Pull Request that introduced the chang
 
 ## main / unreleased
 
+* [CHANGE] Reduce `-server.grpc-max-concurrent-streams` from 1000 to 500 for ingester and to 100 for all components. #5666
```

Comment:
> and to 100 for all components

Is this changed in this PR? I can't se it.

Reply:
The change is because I removed the parameter in the YAML config and set it in the ingester statefulSet. The YAML config used to say `1000` and be used on all components. The CLI args in the ingester statefulSet only apply to the ingester